packages/babel-core/src/transformation/file/options/option-manager.js

```diff
@@ -260,6 +260,17 @@ export default class OptionManager {
       let presetLoc;
       if (typeof val === "string") {
         presetLoc = resolve(`babel-preset-${val}`, dirname) || resolve(val, dirname);
+
+        // trying to resolve @organization shortcat
+        // @foo/es2015 -> @foo/babel-preset-es2015
+        if (!presetLoc) {
+          let matches = val.match(/^(@[^/]+)\/([^/]+)(\/.+)?$/);
```

Comment:
Ah... I forgot update regexp. It should be: `/^(@[^/]+)\/(.+)$/` which splits `@babel/react/optim` to `@babel` and `react/optim`.
You are right, I'll update this now.